lib/dialects/mariadb/connection-manager.js

```diff
@@ -90,7 +90,8 @@ class ConnectionManager extends AbstractConnectionManager {
     return this.lib.createConnection(connectionConfig)
       .then(connection => {
         this.sequelize.options.databaseVersion = connection.serverVersion();
-        debug('connection acquired');
+        connection.uuid = connection.threadId;
+        debug(`connection acquired ${connection.uuid}`);
```

Comment:
`uuid` is assigned when transaction is initialized, so it wont be available here. Just remove these changes from this file

Reply:
ok right, i was confused with sqlite uuid to select connection. commit done to change that